includes/Data/Flows/Flows.php

```diff
@@ -147,6 +126,50 @@ final class Flows {
 		),
 	);
 
+	/**
+	 * Has all the Flow Key parameters to be updated for the user in Key-Value pair.
+	 *
+	 * - the value of old_key i.e OldKey is the key name in the option that has been modified to NewKey in the blueprint
+	 * - the value of new_key i.e NewKey is the key name that is be replaced in the option
+	 * - OldKey should also be manually updated to the NewKey in the blueprint
+	 * - retain_existing_value (Boolean): true if the value in the option for the OldKey is to be retained,
+	 *   false if the NewKey is to have the default value that is set in the blueprint and discard the value set in the option for OldKey
+	 *
+	 * @var array
+	 */
+	protected static $fixes = array(
```

Comment:
Just wanted to ask from now how are we migrating changes? like what is the use-case of `fixes` vs `upgrades`?